google-datacatalog-qlik-connector/tests/google/datacatalog_connectors/qlik/scrape/metadata_scraper_test.py

```diff
@@ -97,12 +102,56 @@ def test_scrape_all_streams_should_return_list_on_success(self):
             'id': 'stream-id',
         }]
 
-        attrs['_MetadataScraper__session'] = \
-            scrape_ops_mocks.FakeSessionWithCookies()
+        attrs['_MetadataScraper__qrs_api_session'] = mock.MagicMock()
         qrs_api_helper.get_full_stream_list.return_value = streams_metadata
 
         streams = self.__scraper.scrape_all_streams()
 
         self.assertEqual(1, len(streams))
         self.assertEqual('stream-id', streams[0].get('id'))
         qrs_api_helper.get_full_stream_list.assert_called_once()
+
+    @mock.patch(f'{__SCRAPER_MODULE}.authenticator.Authenticator')
+    def test_scrape_engine_api_should_authenticate_user(
```

Comment:
Same here, method being tested is `scrape_sheets`, this test case fells out of place, there is already a test suite for authenticator class: `authenticator_test.py`. Perhaps just changing the test case name here too. 

Please let me know your thoughts on this.

Reply:
I've changed the assertions to let it clear the authentication flow is the main thing under test. I'm calling `scrape_sheets` just to traverse the authentication flow. Please let me know whether the intent is clearer now.